### ISSUE TITLE
Not able to add bookmarks

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#583]
```
What steps will reproduce t


What do you see instead?


What version of the product are you using?


On what operating system?


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `ramchand...@gmail.com` on 10 Aug 2013 at 5:04

**Comment by GoogleCodeExporter** [comment]
```
Cannot see 'add bookmark' button.
GT-N8000, android 4.1.2, product version 2.0.8
```

Original comment by `qwertylo...@gmail.com` on 9 Sep 2013 at 12:36

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 1:05
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
still i am facing problems  with ebookdroid on my karbonn smart tab 1, the
pdf file opens, but page is not displayed. Before update it was working
fine.
PL do something.
```

Original comment by `ramchand...@gmail.com` on 4 Jan 2014 at 5:47
